jesd204/jesd204-priv.h

```diff
@@ -0,0 +1,70 @@
+/**
+ * The JESD204 framework
+ *
+ * Copyright (c) 2022 Analog Devices Inc.
+ */
+
+#ifndef _JESD204_PRIV_H_
+#define _JESD204_PRIV_H_
+
+#include "jesd204.h"
+
+#define JESD204_MAX_LINKS	16
+
+/**
+ * struct jesd204_dev - JESD204 device
```

Comment:
I wouldn't do it yet, but it might make sense to update this on both linux and no-OS sometime (since in-line member documentation comments are allowed on Linux too).